specs/linux/http_events.table

```diff
@@ -0,0 +1,21 @@
+table_name("http_events")
+description("Tracks HTTP request header.")
+schema([
+    Column("time", BIGINT, "Time of HTTP event"),
+    Column("method", TEXT, "HTTP request method GET/POST/PUT/DELETE/OPTIONS/PATCH"),
+    Column("protocol", TEXT, "SSL protocol used in communication(only for HTTPS/TLS)"),
```

Comment:
Feels ambiguous. In the context of `http_events`, I don't know if `protocol` means, TLS protocol, http protocol, or something else.